recipes/libssh2/bld.bat

```diff
@@ -0,0 +1,9 @@
+call setup_x64.bat
```

Comment:
Yeah, noticed after. That's fine.

Will definitely work on getting that integrated seamlessly. Thanks for getting it put together. 😉